### DISCUSSION TITLE
New in v2.20.0: Move Tabs & Groups to Other Windows + Planning Features! 🎉

### DISCUSSION THREAD

**Discussion by okaryo** [okaryo/TabTabTab#234] [Announcements]
Hi! 👋

We're excited to announce some exciting updates and upcoming features in [v2.20.0](https://github.com/okaryo/TabTabTab/releases/tag/2.20.0)! Here's what's new and what's on the horizon:

1. New Tab and Tab Group Features in v2.20.0:

    * 🖥️ Move Tabs and Tab Groups to Separate Windows: We've made organizing your workflow even more flexible. Now, you can easily move tabs and tab groups to different windows!

    * https://github.com/okaryo/TabTabTab/assets/44517313/ec70ee4e-355c-4cf6-b5f7-22d09abedb6a

1. Planning Features on the Roadmap:
    * 🔍 Enhanced Tab Search: You'll be able to see recently opened or closed tabs in your tab search results, making it easier to navigate and manage your tabs.
    * 💾 Save and Restore Tab Groups and Window Sessions: We're exploring a feature that would allow you to save your preferred tab groups and window layouts for later use. This potential feature aims to provide a more personalized browsing experience.

1. Share Your Ideas! 🤔
    * 📢 We love hearing from you! If you have any suggestions, ideas, or feedback on what features you'd like to see next, please feel free to share them in the discussions. Your input helps us make the product even better.

Thank you for being a part of our community. We can't wait to bring you these new features and hear your thoughts!

Happy tabbing! 🌐

**Comment by FreePDF64** [comment]
I'm so excited about the announced features!!

Thx!
